pic-build

```diff
@@ -21,6 +21,7 @@
 
 this_dir=`dirname $0`
 build_dir=".build"
+this_dir=$(pwd)/$this_dir
```

Comment:
offline discussed: this is the way how to get the absolute path `this_dir=$(cd `dirname $0` && pwd)`